src/twisted/python/_setup.py

```diff
@@ -96,6 +96,7 @@
         'pyasn1',
         'cryptography >= 0.9.1',
         'appdirs >= 1.4.0',
+        'pynacl >= 1.0.1',
```

Comment:
I should note; it's OK to make the `conch` extra hard depend on this, since for most users, that should be the right thing.  I'm mainly talking about leaving the code in such a state that an escape hatch like a separate `conch-no25519` extra would be possiblel.